### DISCUSSION TITLE
Lolmoney

### DISCUSSION THREAD

**Discussion by MSMAN14** [btcpayserver/btcpayserver#6148] [Technical Support] [answered]
I already pay. I did not receive my product kit.

**Comment by Kukks** [comment, marked as answer]
Contact the merchant directly.

**Reply by MSMAN14** [reply]
I still didn’t even get to use it. I need bank that I could find in with my ATR tool
When I do it and I bring it to the ATM it don’t work it decline

**Reply by MSMAN14** [reply]
Could you send me a new one like a bank that I could find with my ATR tool so I could do my process and complete my process so I could cash out

**Comment by MSMAN14** [comment]
You guys need to send me a new one the one I got not working
